StabilityPackage.wl

```diff
@@ -616,5 +616,21 @@ Return[data];
 ];
 
 
+(*Exports a DOtoMoments dodata file in h5 format.*)
+exportDOasMoments[dodata_,name_]:= 

```

Comment:
I think I'm missing something - this looks like it is exporting DO data in DO format. Where do the moments come in?

Reply:
Yes, this is badly named. The input of this should be the output of DOtoMoments. They way I wrote it, I build the file in the wrong order, defining the association names that ImportFile would assign. The export "undoes" the association, so that when you import the file ImportFile then sees the pre-association names to assign them. It's inelegant.

Reply:
Okay, I would at least change the name to something descriptive and add a comment to explain the complication.